web/packages/teleport/src/AccessRequests/service.ts

```diff
@@ -83,10 +83,12 @@ export async function getDurationOptions(
     return [];
   }
 
-  return middleValues(accessRequest.sessionTTL, accessRequest.maxDuration).map(
-    duration => ({
-      value: duration.timestamp,
-      label: formatDuration(duration.duration),
-    })
-  );
+  return middleValues(
+    accessRequest.created,
+    accessRequest.sessionTTL,
```

Comment:
I don't personally care, but our typescript naming conventions are a bit different than Go. This would typically be named `sessionTtl` in this part of the codebase.